tools/build_defs/pkg/make_deb.py

```diff
@@ -18,7 +18,11 @@
 from io import BytesIO
 import os
 import os.path
-import StringIO
+from io import BytesIO
+try:
+    from StringIO import StringIO
```

Comment:
Looks like Python 2's io module also has a StringIO class. Is there a reason not to replace this try/except with an unconditional `from io import StringIO`?

Reply:
It would need to be a shell integration test under //src/test/shell/bazel. Unfortunately I don't see any existing test coverage (by searching that directory for "pkg_deb"), so unless there's something I'm missing it'd have to be a new file . The idea would be to cargo-cult the existing boilerplate from another file in that directory, then write a test case in shell that writes out the files of a minimal workspace, invokes bazel using Python 2, reads the checksum, then does the same with Python 3 and asserts on the checksum matching.

Getting Bazel to use Python 2 vs Python 3 is a little tricky at the moment. I have a [pending CL](https://bazel-review.googlesource.com/c/bazel/+/83912/15/src/test/shell/bazel/python_version_test.sh#89) to add an integration test for some Python functionality. The setup code for that test, which currently does not work on Mac or Windows, could be reused for the test you'd write. Then you'd do one invocation of Bazel with `--force_python=PY2` and the other with `--force_python=PY3`. Feel free to copy `use_system_python_2_3_runtimes` from my pending CL and then file a bug for me to factor out that helper function into a common util.